core/embed/extmod/modtrezorcrypto/modtrezorcrypto-secp256k1_zkp.h

```diff
@@ -64,8 +65,11 @@ STATIC mp_obj_t mod_trezorcrypto_secp256k1_context_make_new(
   const size_t secp256k1_ctx_size = secp256k1_context_preallocated_size(
       SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY);
 
-  mp_obj_secp256k1_context_t *o = m_new_obj_var_maybe(
-      mp_obj_secp256k1_context_t, uint8_t, secp256k1_ctx_size);
+  // there is no finaliser-enabled "m_new_obj_var_maybe" so we have to call
```

Comment:
Maybe wrapping this into a macro called `m_new_obj_var_maybe_with_finalizer` or something similar would be less cryptic?

Reply:
done in  6fd14cd